docs/pages/writing.vue

```diff
@@ -49,6 +49,9 @@
           Avoid creating new Proper Nouns in casual messages. For example,
           "You are a coach", not "You are a Coach"
         </li>
+        <li>
+          Avoid using gerunds in modal or page titles. This creates the impression that the task is in progress when it may not be. For example, "Publish channel", not "Publishing channel"
```

Comment:
This sort of reads to me like "don't use gerunds incorrectly" :) We might similarly say something like

> Avoid using gerunds in modal or page titles prior to initiating a background task because a gerund communicates that the task is already in progress

but this seems obvious when phrased like that....

I believe @radinamatic had mentioned wanting to avoid gerunds more generally because they tend to cause challenges for translators, but I think that might be a slightly different sort of guidance we're giving above.

Reply:
Noting that it's not a gerund that communicates that the task is in progress, but that in English the gerund form `-ing` and the present participle form `-ing` are identical. So using a gerund `Publishing channel` can easily imply to the reader that this is a contracted form of `The channel is publishing`.

I feel like it would be easier to describe what ought to be done:

> Use imperative verbs in modal or page titles.

Reply:
@radinamatic posted this http://asd-ste100.org/faq.html

![image](https://user-images.githubusercontent.com/2367265/109034814-de263400-767c-11eb-9735-0baf9f0f7df4.png)

But it leaves me wondering (:rimshot:) how we would express things like "Downloading", "Exporting", etc. "The download is in progress" seems so verbose

Reply:
"Use imperative verbs in modal or page titles" is _much_ simpler than what's there now. I'm going to update the guideline with that.

This PR is meant to only address the first point
> 1. this guidance specifically for modal and page titles (in which case I would expect it go on a more specific page) 

I'm thinking I should remove it from the Writing page and add it to the title guidelines in Page headers and Modals only.

For the second point
> 2. in fact there there is blanket avoidance of -ing forms even for things that are actually in-progress?

If we do want to go in that direction, which I'd rather let Radina speak on, I think it's a separate issue that shouldn't be addressed by this PR, and maybe that's the level of guidance that should go in the Writing page.

Reply:
> I'm thinking I should remove it from the Writing page and add it to the title guidelines in Page headers and Modals only.

that would be my preference. otherwise we're sort of bubbling up a one-off writing mistake into a highly-visible new global writing rule